### PR TITLE
i18n(zh-cn): Update resource

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description

- Update resources

#### Upstream
- https://github.com/withastro/starlight/pull/1357

### CODE REVIEW

File: docs/src/content/docs/zh-cn/reference/configuration.mdx [hunk 1]
```diff
@@ -465,7 +465,7 @@ interface HeadConfig {
 
 定义页脚是否应包含上一页和下一页的链接。
 
-页面可以使用 [`prev`](/zh-cn/reference/frontmatter/#prev) 和 [`next`](/zh-cn/reference/frontmatter/#next) frontmatter 字段覆盖此设置或链接文本和/或 URL。
+页面可以使用 [`prev`](/zh-cn/reference/frontmatter/#prev) 和 [`next`](/zh-cn/reference/frontmatter/#next) frontmatter 字段覆盖此设置或链接文本 **和/或** URL。
```
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/reference/configuration.mdx:468]: Is this change related to a specific commit? I don't see bolded text in the English version so I was wondering what was the reasoning for the chamge.

File: docs/src/content/docs/zh-cn/reference/configuration.mdx
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/reference/configuration.mdx:1]: I cannot add a comment for this, but the link to the plugin section on the plugin page in the [`plugins` option description](https://starlight-git-fork-liruifengv-patch-0123-astrodotbuild.vercel.app/zh-cn/reference/configuration/#plugins) is missing its anchor.

It should link to `/zh-cn/resources/plugins/#插件` (the `#插件` is missing).

File: docs/src/content/docs/zh-cn/resources/plugins.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		href="https://github.com/HiDeoo/starlight-blog"
+		title="starlight-blog"
+		description="添加一个博客到你的文档网站。"
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-openapi"
+		title="starlight-openapi"
+		description="从 OpenAPI/Swagger 规范创建文档页面。"
+	/>
+</CardGrid>
```
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/resources/plugins.mdx:64]: This is missing the `notion-to-astro` and `astro-live-code` cards from https://starlight.astro.build/resources/plugins/#community-tools-and-integrations

File: docs/src/content/docs/zh-cn/resources/showcase.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Starlight Showcase
```
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/resources/showcase.mdx:2]: Was this not translated on purpose? Or maybe just forgot?

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few questions and comments but overall, this looks really good to me. :thumbsup:

File: docs/src/content/docs/zh-cn/resources/showcase.mdx (hunk 3 above)
Comment (liruifengv) [reply docs/src/content/docs/zh-cn/resources/showcase.mdx]: Yeah. I updated it.

File: docs/src/content/docs/zh-cn/reference/configuration.mdx (hunk 1 above)
Comment (liruifengv) [reply docs/src/content/docs/zh-cn/reference/configuration.mdx]: Thanks. I restored it.

**Comment by liruifengv** [comment]
> Thanks for the translation 🙌
> 
> As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few questions and comments but overall, this looks really good to me. 👍

Thanks for your review. I have updated them.

File: docs/src/content/docs/zh-cn/resources/community-content.mdx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	/>
+	<LinkCard
+		href="https://blog.otterlord.dev/posts/starlight-sponsors/"
+		title="将赞助商添加到你的 Starlight 网站"
+		description="在文档侧边栏中实现自定义赞助商组件的指南"
+	/>
+	<LinkCard
+		href="https://hideoo.dev/notes/starlight-og-images"
+		title="将 Open Graph 图像添加到 Starlight"
+		description="A guide to generating social images and the corresponding meta tags for your pages"
```
Comment (delucis) [inline docs/src/content/docs/zh-cn/resources/community-content.mdx:60]: I think this line was missed for translation.

**Review by delucis — COMMENTED** [review]
Thanks for the careful translation @liruifengv — all looks great! I just spotted one line that I think still needs updating.

File: docs/src/content/docs/zh-cn/resources/community-content.mdx (hunk 4 above)
Comment (liruifengv) [reply docs/src/content/docs/zh-cn/resources/community-content.mdx]: Fixed it.

**Review by delucis — APPROVED** [review]
Perfect — thank you again @liruifengv! 🌟
